### DISCUSSION TITLE
[Proposal] changing the md5 to the xxh128 in Laravel 10 ?

### DISCUSSION THREAD

**Discussion by kitrio** [laravel/framework#46074] [General] [answered]
xxh128 hash function is faster than md5. xxh128 is supported from PHP 8.1 onwards

Laravel 10 is supprot PHP 8.1

Recently, xxh128 has also been applied to Symfony 6.3  commit

here is xxh128 benchmark 
[https://php.watch/versions/8.1/xxHash](
https://php.watch/versions/8.1/xxHash)

xxh128 is 14 times faster than md5. This helps with improved cache performance

**Comment by JurianArie** [comment, marked as answer]
See https://github.com/laravel/framework/commit/199772943310e93b93f42b1c6d38abd6cad527df, xxh128 will be used for blade in Laravel 10.
